Tasks/ServiceFabricDeployV1/Update-DockerSettings.psm1

```diff
@@ -57,7 +57,7 @@ function Update-DockerSettings
             }
         }
 
-        if (-not $isPasswordEncrypted -and $ClusterConnectionParameters["ServerCertThumbprint"])
+        if (-not $isPasswordEncrypted -and ($ClusterConnectionParameters["ServerCertThumbprint"] -or $ClusterConnectionParameters["ServerCommonName"]))
```

Comment:
I don't think $ClusterConnectionParameters will contain thumbprint or commonname properties. these are filled up locally in Connect-ServiceFabricClusterFromServiceEndpoint and hence won't be available in deploy.ps1 which is the caller here. You will need to use service endpoint here.

Reply:
My understanding was Connect-ServiceFabricClusterFromServiceEndpoint properly initializes these values?
@bishal-pdMSFT @KirillPa ?

Reply:
Values initialized by Connect-ServiceFabricClusterFromServiceEndpoint will be available in this file as well.